src/components/searchForm/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { Button, Input } from "..";
+
+export const SearchForm = () => {
+  const onSearch = () => {
+    // TODO: Finalise search action w/ api call / filter
+  };
+
+  return (
+    <div>
+      <Input />
+      <Button onClick={() => onSearch()}>Search</Button>
```

Comment:
No need to pass a callback to call onSearch if you have no arguments to pass to the function.